GOVERNANCE.md

```diff
@@ -0,0 +1,103 @@
+## Governance
+
+This OCA Bundle registry is managed by the [BC Government Digital Trust Group].
+The contents of the registry is maintained in the [registry repository] and
+managed through a pull request submission, review and approval process. The
+details of how pull requests are managed is outlined in this Governance section
+of the registry.
+
+[BC Government Digital Trust Group]: https://digital.gov.bc.ca/digital-trust/
```

Comment:
In the future, this "group" will need better definition. Who is this? What governance model do the people operate under? How are merging decisions decided? Contact info.

Reply:
I was hoping you had some ideas on that, in both this case add in the larger cases we’d like to use this — Pan Canadian and even Aries Community.

I could have put in “Senior Leadership of the Digital Trust Group”, or I could have put “Whatever Jill says”.  Is there some useful wording to put into here to describe the group?

My challenge with this is that as humans we know what I mean by that, and my claims is that it's only in extreme edge cases where a precise, “stand up in court” definition is needed.  How far do we need to go to cover these edge cases?

In the Hyperledger case, I put in something like a dispute mechanism, and where it should go. E.g. “In case of dispute about a PR that is not merged, the Contributor should contact Hyperledger Technical Oversight Committee for resolution.”  That could be done here as well, but it definitely starts to tred into legal territory, which I think should be avoided.

An alternative.  We could add a clause that says, “The Digital Trust Group offers this repository and the services of the Editors. In return, it reserves the right to accept or reject an Pull Request submitted to this repository without providing a reason.”  Would that help.